.github/workflows/update-registry.yml

```diff
@@ -74,12 +87,12 @@ jobs:
             
             - Added chain ID ${{ inputs.chain_id }} to all contracts in version ${{ inputs.version }}
           title: |
-            Add chain ${{ inputs.chain_id }} to ${{ inputs.version }} registry
+            Add chain ${{ inputs.chain_id }} to ${{ inputs.version == 'v1.3.0-both' && 'v1.3.0 (canonical + eip155)' || inputs.version }} registry
           body: |
             This PR adds chain ID `${{ inputs.chain_id }}` to the registry for version `${{ inputs.version }}`.
```

Comment:
I didn't notice before, but noticed now that PR body doesn't follow our PR template that is required for CI to pass, so networks introduced this way to not pass CI.